code/modules/clothing/suits/wintercoats.dm

```diff
@@ -419,3 +419,21 @@
 	desc = "A dusty winter coat hood."
 	icon_state = "hood_miner"
 	armor = list(MELEE = 10, BULLET = 0, LASER = 0, ENERGY = 0, BOMB = 0, BIO = 0, RAD = 0, FIRE = 0, ACID = 0)
+
+/obj/item/clothing/suit/hooded/wintercoat/custom
+	name = "tailored winter coat"
+	desc = "A heavy jacket made from 'synthetic' animal furs, with custom colors."
+	greyscale_colors = "#ffffff#ffffff#808080#808080#808080#808080"
+	greyscale_config = /datum/greyscale_config/winter_coats
+	greyscale_config_worn = /datum/greyscale_config/winter_coats/worn
+	hoodtype = /obj/item/clothing/head/hooded/winterhood/custom
+
+/obj/item/clothing/suit/hooded/wintercoat/custom/MakeHood()
+	. = ..()
+	hood.set_greyscale(copytext(greyscale_colors, 1, 22)) //Adopt the suit's grayscale coloring for visual clarity.
```

Comment:
Use `SSgreyscale.ParseColorString()` and do a Copy() to get the entries you want instead of this, things like `#123` and `#11223344` are also valid colors.